test/unit/render/glyph_manager.test.js

```diff
@@ -28,6 +28,36 @@ test('GlyphManager requests 0-255 PBF', (t) => {
     });
 });
 
+test('GlyphManager doesn\'t request twice 0-255 PBF if a glyph is missing', (t) => {
```

Comment:
Thanks a lot for your pull request. This change looks great. Just a small suggestion here, since this test and `GlyphManager requests 0-255 PBF` test are using a lot of similar code, I would suggest using helper functions in order to avoid repeating the same code.

Reply:
Done